bot/cogs/error_handler.py

```diff
@@ -87,9 +88,29 @@ def __init__(self, bot: Bot):
                     return
 
                 # Return to not raise the exception
-                with contextlib.suppress(ResponseCodeError):
-                    await ctx.invoke(tags_get_command, tag_name=ctx.invoked_with)
```

Comment:
Is suppressing this exception no longer necessary? Maybe not since the tag suggestion feature, but not sure.